stories/Map.tsx

```diff
@@ -26,11 +30,27 @@ storiesOf('Atoms|Map', module)
                 },
                 radius: 30000,
             };
+            const pointMarker = {
+                center: {
+                    lat: 52.5112671,
+                    lng: 7.2535521,
+                },
+            };
             const markers = {
                 marker1: [defaultMarker],
                 marker2: [addedMarker],
                 multiple: [defaultMarker, addedMarker],
                 none: [],
+                'pointer marker': [pointMarker],
```

Comment:
zoom for point-marker doesn't work well. But anyway, it is not our usecase. Maybe better remove it from the examples.

Reply:
Actually, I added it for purpose. After my updates a couple of tests were failing and I was wondered why it happens when we add marker without radius. So, even considering that the zoom does not work properly I would like to keep it, because it might be helpful for debug and demonstration purposes.